packages/flutter_tools/templates/create/projectName.iml.tmpl

```diff
@@ -11,5 +11,6 @@
     <orderEntry type="sourceFolder" forTests="false" />
     <orderEntry type="library" name="Dart SDK" level="application" />
```

Comment:
This will make the template work w/ both 2016.3 and 2017.1; a 2017.1 IntelliJ will remove the `'name="Dart SDK" level="application"'` line when it opens the project.